code/datums/antagonists/datum_vampire.dm

```diff
@@ -0,0 +1,280 @@
+/datum/antagonist/vampire
+	var/usable_blood = 0
+	var/total_blood = 0
+	var/fullpower = FALSE
+	var/draining
+	var/list/objectives_given = list()
+
+	var/iscloaking = FALSE
+
+	var/list/powers = list() // list of current powers
+
+	var/obj/item/clothing/suit/draculacoat/coat
+
+	var/list/upgrade_tiers = list(

```

Comment:
Have you adjusted these numbers to account for human having like 500 u of blood apiece?

Reply:
yeah i adjusted it so you wouldn't get a shitton of powers for succing one or two nerds